config.md

```diff
@@ -323,7 +328,7 @@ For Windows based systems the user structure has the following fields:
 ## <a name="configPlatformSpecificConfiguration" />Platform-specific configuration
 
 [**`platform.os`**](#platform) is used to specify platform-specific configuration.
-Runtime implementations MAY support any valid values for platform-specific fields as part of this configuration.
+Runtimes MAY support any valid values for platform-specific fields as part of this configuration.
```

Comment:
> ... I'd change the three 'Runtimes' below to that too.

Would you update [the other instances][1] (beyond those already touched by this PR) too?

Can you put a finger on why you likw including "implementations" better?  A bare "runtimes" is shorter, and I can't think of a non-implementation runtime.

[1] :https://github.com/opencontainers/runtime-spec/pull/811#pullrequestreview-38088236